.editorconfig

```diff
@@ -14,7 +14,7 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 end_of_line = lf
-max_line_length = null
+max_line_length = 9999
```

Comment:
I think it was pytest? that wanted this to either `== off` or be a int – but off didn't work.